gitlab/datadog_checks/gitlab/data/conf.yaml.example

```diff
@@ -354,22 +403,145 @@ init_config:
     #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
     #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
     #   no_proxy:
-    #     - <HOSTNAME_1>
-    #     - <HOSTNAME_2>
+    #   - <HOSTNAME_1>
+    #   - <HOSTNAME_2>
 
     ## @param skip_proxy - boolean - optional - default: false
     ## If set to `true`, this makes the check bypass any proxy
     ## settings enabled and attempt to reach services directly.
     #
     # skip_proxy: false
 
-## Log Section (Available for Agent >=6.0)
+    ## @param timeout - number - optional - default: 10
+    ## The timeout for connecting to services.
+    #
+    # timeout: 10
+
+    ## List of legacy metrics to extract from Prometheus.
```

Comment:
Is this param missing type?
```
  ## @param allowed_metrics - list of strings - optional
  ## List of legacy metrics to extract from Prometheus.
  #
```

Reply:
Yes- I had the type as `list` instead of `array`